README.md

```diff
@@ -1,2 +1,10 @@
+##pyladies meetup
+####Clover Health [Map](https://goo.gl/maps/B6mkXhhjMMH2)
+######Date:22 Sept 2016 - Session 2of2
+######Time:6-8:00pm
```

Comment:
Since this is an example repo I'll be using in the future, I'm trying not to tie it to a particualr date or location.  Thanks tho ;)